docs/getting-started/quickstart.md

```diff
@@ -4,31 +4,38 @@ In this tutorial we will
 
 1. Deploy a local feature store with a **Parquet file offline store** and **Sqlite online store**.
 2. Build a training dataset using our time series features from our **Parquet files**.
-3. Materialize feature values from the offline store into the online store.
-4. Read the latest features from the online store for inference.
-
-You can run this tutorial in Google Colab or run it on your localhost, following the guided steps below.
-
-![](../.gitbook/assets/colab\_logo\_32px.png)[**Run in Google Colab**](https://colab.research.google.com/github/feast-dev/feast/blob/master/examples/quickstart/quickstart.ipynb)
```

Comment:
It's unlikely to stay up to date because Colab is indefinitely fixed to Python 3.7. If we wanted something like this, using http://mybinder.org might work, but that also has its cons

Reply:
I also imagine that we actually want GitHub actions to be included too, which would make more sense to handle locally rather than in Colab.